io.openems.edge.meter.schneider.acti9.smartlink/src/io/openems/edge/meter/schneider/acti9/smartlink/MeterSchneiderActi9Smartlink.java

```diff
@@ -99,20 +100,27 @@ protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
 						m(AsymmetricMeter.ChannelId.VOLTAGE_L3, new FloatDoublewordElement(3032 - offset),
 								ElementToChannelConverter.SCALE_FACTOR_3)),
 				new FC4ReadInputRegistersTask(3054 - offset, Priority.HIGH,
-						m(AsymmetricMeter.ChannelId.ACTIVE_POWER_L1, new FloatDoublewordElement(3054 - offset)),
-						m(AsymmetricMeter.ChannelId.ACTIVE_POWER_L2, new FloatDoublewordElement(3056 - offset)),
-						m(AsymmetricMeter.ChannelId.ACTIVE_POWER_L3, new FloatDoublewordElement(3058 - offset)),
-						m(SymmetricMeter.ChannelId.ACTIVE_POWER, new FloatDoublewordElement(3060 - offset)),
+						m(AsymmetricMeter.ChannelId.ACTIVE_POWER_L1, new FloatDoublewordElement(3054 - offset),
+								ElementToChannelConverter.INVERT_IF_TRUE(inverted)),
```

Comment:
Our Checkstyle file requires writing `this.inverted`. Unfortunately using Checkstyle is not yet documented in OpenEMS. The definition file is here (https://github.com/OpenEMS/openems/blob/develop/cnf/checkstyle.xml) and can be opened in Eclipse using the Checkstyle addon.

Reply:
Thanks, I usually do this. as well. Missed it for these ones. I added the checkstyle config to my eclipse and updated the codebase.

Reply:
Thanks. Unfortunately adding Checkstyle to the .project file causes errors in the IDE when the addon is not installed. I want to avoid this error for newcomers, so I'll restore the .project file.